buildh/UI.py

```diff
@@ -101,13 +101,15 @@ def parse_cli():
                         help="The first frame (ps) to read from the trajectory.")
     parser.add_argument("-e", "--end", type=int,
                         help="The last frame (ps) to read from the trajectory.")
+    parser.add_argument("-ich", "--ignore-CH3s", action='store_true',
```

Comment:
Woud that be possible to call the short flag version `-igch3` ?

Reply:
yes, it's a better name!